src/components/renderer/page/page.renderer.component.tsx

```diff
@@ -42,25 +55,51 @@ function PageRenderer({ page }: PageRendererProps) {
               </span>
             </p>
           </div>
-          {!isCollapsed && (
+          <div className={isCollapsed ? styles.hiddenAccordion : styles.accordionContainer}>
             <Accordion>
-              {visibleSections.map((section) => (
-                <AccordionItem
-                  title={t(section.label)}
-                  open={true}
-                  className={styles.sectionContainer}
-                  key={`section-${section.label}`}>
-                  <div className={styles.formSection}>
-                    <SectionRenderer section={section} />
-                  </div>
-                </AccordionItem>
+              {visibleSections.map((section, index) => (
```

Comment:
Unrelated to your changes, but maybe `visibleSections` could benefit from memoization:

```tsx
const visibleSections = useMemo(() => {
  return page.sections.filter((section) => {
    const hasVisibleQuestions = section.questions.some((question) => !isTrue(question.isHidden));
    return !isTrue(section.isHidden) && hasVisibleQuestions;
  });
}, [page.sections]);
```